package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "build": "tsc -p tsconfig.json --outdir lib",
-    "prepare": "npm run build",
+    "prepublish": "npm run build",
```

Comment:
Downstream CI has been failing since this package switched to TypeScript. I suggested the change as a way to fix that.

Note that CI is using npm@3 and `prepare` support was only added in v4.

Reply:
I see